mlflow/entities/model_registry/model_version.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         self._status = status
         self._status_message = status_message
         self._tags = {tag.key: tag.value for tag in (tags or [])}
-        self._aliases = aliases
+        self._aliases = aliases or []
```

Comment:
Line 199 in this file calls extend on `self.aliases`:

```
model_version.aliases.extend(self.aliases)
```

This doesn't work if `self.aliases` is `None`. `model_version.aliases` is an object in protobuf that represents a repeated field. In an older version of protobuf, `extend(None)` doesn't throw, but in newer versions, it does. Noticed this issue in https://github.com/mlflow/mlflow/actions/runs/4637819736/jobs/8207058006.

Reply:
nice finding!